test/e2e/e2e_test.go

```diff
@@ -465,513 +496,165 @@ var _ = Describe("test e2e inter-node connectivity between worker nodes hybrid o
-		if err != nil {
-			framework.Failf("Expected container %s running on %s error %v", ovnContainer, ovnWorkerNode, err)
-		}
-		if kubectlOut == "''" {
-			haMode = true
-			kubectlOut, err = framework.RunKubectl("get", "pods", ovnNsFlag, "-l", labelFlag, jsonFlag, fieldSelectorHaFlag)
-			if err != nil {
-				framework.Failf("Expected container %s running on %s error %v", ovnContainer, ovnHaWorkerNode2, err)
-			}
-		}
-		// Fail the test if no pod is matched within the specified node
-		if kubectlOut == "''" {
-			framework.Failf("Unable to locate container %s on any known nodes", ovnContainer)
-		}
-	})
-
-	It("Should validate connectivity within a namespace of pods on separate nodes", func() {
-		var err error
-		var validIP net.IP
-		var pingTarget string
-		var ciWorkerNodeSrc string
-		var ciWorkerNodeDst string
-		dstPingPodName := "e2e-dst-ping-pod"
-		command := []string{"bash", "-c", "sleep 20000"}
-		// non-ha ci mode runs a named set of nodes with a prefix of ovn-worker
-		ciWorkerNodeSrc = ovnWorkerNode
-		ciWorkerNodeDst = ovnWorkerNode2
-		// ha ci mode runs a named set of nodes with a prefix of ovn-control-plane
-		if haMode {
-			framework.Logf("Detected a HA mode KIND environment")
-			ciWorkerNodeSrc = ovnHaWorkerNode2
-			ciWorkerNodeDst = ovnHaWorkerNode3
-		}
-		By(fmt.Sprintf("Creating a container on node %s and verifying connectivity to a pod on node %s", ciWorkerNodeSrc, ciWorkerNodeDst))
-
-		// Create the pod that will be used as the destination for the connectivity test
-		createGenericPod(f, dstPingPodName, ciWorkerNodeDst, command)
-
-		// There is a condition somewhere with e2e WaitForPodNotPending that returns ready
-		// before calling for the IP address will succeed. This simply adds some retries.
-		for i := 1; i < getPodIPRetry; i++ {
-			pingTarget, err = getPodAddress(dstPingPodName, f.Namespace.Name)
-			if err != nil {
-				framework.Logf("Warning unable to query the test pod on node %s %v", ciWorkerNodeSrc, err)
-			}
-			validIP = net.ParseIP(pingTarget)
-			if validIP != nil {
-				framework.Logf("Destination ping target for %s is %s", dstPingPodName, pingTarget)
-				break
-			}
-			time.Sleep(time.Second * 4)
-			framework.Logf("Retry attempt %d to get pod IP from initializing pod %s", i, dstPingPodName)
-		}
-		// Fail the test if no address is ever retrieved
-		if validIP == nil {
-			framework.Failf("Warning: Failed to get an IP for target pod %s, test will fail", dstPingPodName)
-		}
-		// Spin up another pod that attempts to reach the previously started pod on separate nodes
-		framework.ExpectNoError(
-			checkConnectivityPingToHost(f, ciWorkerNodeSrc, "e2e-src-ping-pod", pingTarget, ipv4PingCommand, 30, false))
-	})
-})
-
-// Verify pods in the namespace annotated with an external-gateway traverse the vxlan
-// overlay and reach the intended external gateway vtep and gateway end to end
-var _ = Describe("e2e external gateway validation", func() {
```

Comment:
this is duplicate with the multiple e2e external gateway,

Reply:
> this is duplicate with the multiple e2e external gateway,

You say this but then the test is not actually removed... did you decide this one was not a duplicate?

Reply:
my bad, the comment is showing up in two different commits